schema.graphql

```diff
@@ -102,3 +102,18 @@ type MarginChangedEvent @entity {
   blockNumber: BigInt! # last updated block number
   timestamp: BigInt! # last updated timestamp
 }
+
+type Trader @entity {
+  id: ID!
+  dayData: [TraderDayData!] @derivedFrom(field: "trader")
+}
```

Comment:
`Trader` could be merged into `Position` as they both have 1:1 relationship with a trader's address

Reply:
Hmm, I think it is better to keep is separate just for the sake of understandability

Reply:
fair enough, let's add a relationship to `Position` then so we don't have to make two queries to get both:
```
position: Position!
```

Reply:
done